ts/features/lollipop/saga/index.ts

```diff
@@ -19,10 +19,20 @@ export function* generateLollipopKeySaga() {
     const newKeyTag = uuid();
     yield* put(lollipopKeyTagSave({ keyTag: newKeyTag }));
     yield* cryptoKeyGenerationSaga(newKeyTag, maybeOldKeyTag);
-    if (!lollipopLoginEnabled) {
```

Comment:
Are we sure we want to remove the check on the lollipop FF ?

Reply:
Good point. But this banner doesn't pertain to the login with lollipop (local FF) that it should be set to TRUE in another PR that pairs with this one. But here we are not checking `isLollipopEnabledSelector` anymore 🤔. Do you think it should be better to leave the remote flag check?

Reply:
Should we also remove the local feature flag and use just the remote one, now that the backend is in production mode?

Reply:
You both anticipated my suggestion of changes 😄

I would suggest at least to remove the local one if we don't use it anywhere